paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/activity/LaunchPaymentSheetCompleteActivityExposePaymentSheetArgs.kt

```diff
@@ -0,0 +1,57 @@
+package com.stripe.android.paymentsheet.example.activity
+
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.compose.setContent
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.platform.LocalContext
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.PaymentSheetContract
+
+internal class LaunchPaymentSheetCompleteActivityExposePaymentSheetArgs :
+    BasePaymentSheetActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            MaterialTheme {
+                val stripeLauncher =
+                    rememberLauncherForActivityResult(contract = PaymentSheetContract())
+                    {
+                        onPaymentSheetResult(it)
+                    }
+
+                val inProgress by viewModel.inProgress.observeAsState(false)
+                val status by viewModel.status.observeAsState("")
+
+                if (status.isNotBlank()) {
+                    Toast.makeText(LocalContext.current, status, Toast.LENGTH_SHORT).show()
```

Comment:
Nit: can we use snackbar instead?

Reply:
This is what we have been using, we can think about changing in the future